src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -251,6 +251,7 @@ public function get_items( $request ) {
 			'search'         => 's',
 			'slug'           => 'post_name__in',
 			'status'         => 'post_status',
+			'exact_search'   => 'exact',
```

Comment:
These appear to be organised alphabetically by key, so the new `exact_search` key should be placed above `exclude`.

Reply:
Thanks @mehulkaklotar!